src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -822,7 +821,13 @@ internal void WriteInformation(InformationRecord record, bool overrideInquire =
                         }
                     }
                 }
-                    
+
+                // Both informational and PSHost-targeted messages are transcribed here.
+                // The only difference betwen these two is that PSHost-targeted messages are transcribed
+                // even if InformationAction is SilentlyContinue.
+                // Previously messages targetting PSHost were transcribed in InternalUI.Write* methods.
+                // Because of that for -InformationAction Inquire the order of written message and choice prompt
+                // was different in host output and transcript file.
```

Comment:
@mklement0 Could you please help and reword the comment?

Reply:
@iSazonov: It sounds good to me in general, is there a specific aspect you're not happy with / that you think is missing?

On a copy-editing note, I suggest placing commas after "Previously" and "Because of that" and perhaps say "the  output sequencing of the message and the prompt choices differed between the host and the transcript".

Taking a step back:  we could put the information about how it _used to_ work in the commit message rather than  in a code comment.

Reply:
@mklement0 Thanks! (English is not my native)
@hubuk Olease address @mklement0 comments (and update PR description) and correct typo in "betwen".

Reply:
Done